plaid/src/apis/npm/npm_web_client.rs

```diff
@@ -254,11 +261,7 @@ impl Npm {
                 .clone()
                 .map_or("Read and write".to_string(), |v| v.to_string()),
             selected_orgs: specs.selected_orgs.clone().unwrap_or(vec![]),
-            selected_packages: params
-                .specs
-                .selected_packages
-                .clone()
-                .unwrap_or(vec![scoped_package]),
+            selected_packages: scoped_packages,
```

Comment:
We take packages as an explicit parameter, so we must use those when defining the scope. It did not make sense that the token specs can contain an entirely different list of packages.